### PR TITLE
feat(release): Add release summary UI

### PR DESCRIPTION
**Pull request by tuannn2**

### CODE REVIEW

File: src/app/[locale]/components/add/components/AddComponents.module.css
Comment (hoangnt2) [inline src/app/[locale]/components/add/components/AddComponents.module.css:1]: move css file to same location as page.tsx

File: src/components/components/GeneralInfoComponent.tsx
Comment (hoangnt2) [inline src/components/components/GeneralInfoComponent.tsx:1]: Remove components and releases folder.
Create new folder GeneralInfoComponent and move this file into it. Do the same things with other file

File: src/app/[locale]/components/add/components/AddComponents.module.css
Comment (tuannn2) [reply src/app/[locale]/components/add/components/AddComponents.module.css:1]: Done

File: src/components/components/GeneralInfoComponent.tsx
Comment (tuannn2) [reply src/components/components/GeneralInfoComponent.tsx:1]: Done

**Review by hoangnt2 — APPROVED** [review]

**Pull request merged**